.github/ISSUE_TEMPLATE/ci-update.md

```diff
@@ -0,0 +1,29 @@
+---
+name: "\U0001F40B Update CI Docker Image"
+about: Provide information on CI Docker Images requiring updates
+title: "[CI] "
```

Comment:
I would suggest using other tags like `[Docker]` or something else for updating CI docker images, because `[CI]` looks more like an CI issue (e.g., flaky test, machine down, etc) at the first glance. Since CI issue is not really a bug, we could still have a CI issue template to let users report CI running issues.

Reply:
On the other hand, [Docker] is a confusing name too, which might sound like docker-related but it’s actually CI upgrade 😆

Reply:
What about [CI Image]

Reply:
I was inclined to suggest we limit the amount of issue types but then this PR failed CI, so I've changed this to `[CI Image] ` and created a `[CI Issue] ` template as well. I don't think that's too many for people to pick through?

It's also worth noting we can use `label` as well as the `title` here if we wanted to create labels for filtering on, I would need someones help to make appropriate labels though.

Reply:
`[CI Image]` looks good to me. `[CI Issue]` looks a bit fanny tho, as we are already creating an "issue" here lol
I'm fine if others feel CI failures are a kind of bugs. The downside is I don't think people can add labels to an issue by themselves unless they have the write permission (i.e., committer). It means we will need a committer to help add labels manually for created issues; otherwise we will need a bot to automatically add labels based on the issue content.

Reply:
By the power of the thesaurus I have renamed it `CI Problem`, though keen to hear whether people think we need another category.

On the labels point, the config appears to let me set labels, so I'm curious if it'll actually apply them when we open those issue types. When we see if that works we can decide if we want to try more of it?

Reply:
I'm not sure either. Maybe someone else could confirm? Anyways, I'm good with the current change.

Reply:
Following up on this thread, a new bug was raised (https://github.com/apache/tvm/issues/8925) which had the label auto-applied so the config here overrides the need for a committer :tada: